Vue-Basic-Video-Chat/src/components/Publisher.vue

```diff
@@ -3,10 +3,9 @@
 </template>
 
 <script>
-import OT from '@opentok/client';
-
+import OT from "@opentok/client";
```

Comment:
Here and throughout, I see you've changed double quotes to single quotes. But we are using AirBnB ES5 rules (see the .eslintrc at the root of the project), so can we change these back to single quotes (to match the other samples)?

Reply:
I see that if you run `npm run lint` at the root of this app, it doesn't complain about the double quotes. But let's use single quotes to minimize the changes in this PR and to match the AirBnB style used by the other apps.

Reply:
@jeffswartz, apparently I had some kind of VS Code extension that was changing those.  Pushed a commit restoring the single quotes.